src/pages/Exchange.page.tsx

```diff
@@ -9,28 +9,22 @@ type Props = {
 }
 
 const ExchangePage: FC<Props> = ({exchangeData}) => {
-    const [carrency, setCarrency] = useState('USD');
-    const [selectedOption, setSelectedOption] = useState("NBU")
+    // const [carrency, setCarrency] = useState('USD');
+    // const [selectedOption, setSelectedOption] = useState("NBU")
 
 
-    const handleChange = (event: SelectChangeEvent) => {
-        setCarrency(event.target.value as string);
-    };
+    // const handleChange = (event: SelectChangeEvent) => {
+    //     setCarrency(event.target.value as string);
+    // };
 
-    const handleChangeProvider = (event: SelectChangeEvent) => {
-        setSelectedOption(event.target.value);
-    };
+    // const handleChangeProvider = (event: SelectChangeEvent) => {
```

Comment:
For comments is the same as on backend.
But if need it in a future - leave it

Reply:
fixed: deleted these comments